geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala

```diff
@@ -0,0 +1,366 @@
+
+    val authorizationsProvider = AccumuloStoreHelper.getAuthorizationsProvider(auths.split(","), conn)
+
+    val rasterStore = new AccumuloRasterStore(conn, tableName, authorizationsProvider, writeVisibilities,
+                        writeMemoryConfig, writeThreadsConfig, queryThreadsConfig, collectStats)
+    // this will actually create the Accumulo Table
+    rasterStore.createTableStructure()
+
+    rasterStore
+  }
+
+  def apply(config: JMap[String, Serializable]): AccumuloRasterStore = {
+    val username: String     = userParam.lookUp(config).asInstanceOf[String]
+    val password: String     = passwordParam.lookUp(config).asInstanceOf[String]
+    val instance: String     = instanceIdParam.lookUp(config).asInstanceOf[String]
+    val zookeepers: String   = zookeepersParam.lookUp(config).asInstanceOf[String]
+    val auths: String        = authsParam.lookupOpt[String](config).getOrElse("")
+    val vis: String          = visibilityParam.lookupOpt[String](config).getOrElse("")
+    val tablename: String    = tableNameParam.lookUp(config).asInstanceOf[String]
+    val useMock: Boolean     = mockParam.lookUp(config).asInstanceOf[String].toBoolean
+    val wMem: Option[String] = writeMemoryParam.lookupOpt[String](config)
+    val wThread: Option[Int] = writeThreadsParam.lookupOpt[Int](config)
+    val qThread: Option[Int] = queryThreadsParam.lookupOpt[Int](config)
+    val cStats: Boolean      = java.lang.Boolean.valueOf(statsParam.lookupOpt[Boolean](config).getOrElse(false))
+
+    AccumuloRasterStore(username, password, instance, zookeepers, tablename,
+      auths, vis, useMock, wMem, wThread, qThread, cStats)
+  }
+
+  val boundsCache =
+    CacheBuilder.newBuilder()
+      .expireAfterAccess(10, TimeUnit.MINUTES)
+      .expireAfterWrite(10, TimeUnit.MINUTES)
+      .build[String, ImmutableSetMultimap[Double, Int]]
+}
+
+object AccumuloRasterTableConfig {
+  /**
+   * documentation for raster table settings:
+   *
+   * table.security.scan.visibility.default
+   * - The default visibility for the table
+   *
+   * table.iterator.majc.vers.opt.maxVersions
+   * - Versioning iterator setting
+   * - max versions, major compaction
+   *
+   * table.iterator.minc.vers.opt.maxVersions
+   * - Versioning iterator setting
+   * - max versions, minor compaction
+   *
+   * table.iterator.scan.vers.opt.maxVersions
+   * - Versioning iterator setting
+   * - max versions, scan time
+   *
+   * table.split.threshold
+   * - The split threshold for the table, when reached
+   * - Accumulo splits the table into tablets of this size.
+   *
+   * @param visibilities
+   * @return
+   */
+  def settings(visibilities: String): Map[String, String] = Map (
+    "table.security.scan.visibility.default" -> visibilities,
+    "table.iterator.majc.vers.opt.maxVersions" -> "1",
+    "table.iterator.minc.vers.opt.maxVersions" -> "1",
+    "table.iterator.scan.vers.opt.maxVersions" -> "1",
+    "table.split.threshold" -> "512M"
```

Comment:
This is another default value which at some point might go elsewhere. This one is buried....